Source/Cryptlex.Net/Requests/Releases/CreateReleaseData.cs

```diff
@@ -1,25 +1,34 @@
-﻿namespace Cryptlex.Net.Releases
+using System.Text.Json.Serialization;
+namespace Cryptlex.Net.Releases
 {
     public class CreateReleaseData
     {
-        public string name { get; set; }
-        public string version { get; set; }
-        public string channel { get; set; }
-        public string platform { get; set; }
+		[JsonPropertyName("name")]
+        public string Name { get; set; }
+		[JsonPropertyName("version")]
+        public string Version { get; set; }
+		[JsonPropertyName("channel")]
+        public string Channel { get; set; }
+		[JsonPropertyName("platform")]
+        public string Platform { get; set; }
+		[JsonPropertyName("@private")]
         public bool? @private { get; set; }
```

Comment:
Property name serialized should be "private"
https://api.cryptlex.com/v3/docs#tag/Releases/operation/post/v3/releases

```
[JsonPropertyName("private")]
public bool? Private { get;  set; }
```